treerec/tskit/parallel_sort.hpp

```diff
@@ -0,0 +1,13 @@
+#ifdef __cplusplus
+#include <tskit/tables.h>
+extern "C"{
+#endif 
+
+
+template<typename T>
```

Comment:
Is this template live code?  It looks like you've now got separate implementations for psort_mutations(), psort_sites(), etc.; but I'm not sure I'm fully understanding the diffs.  If this is obsolete code, please clear it out.

Reply:
Done!